tests/testthat/test-model-tree.R

```diff
@@ -346,6 +370,84 @@ withr::with_options(list(bbr.bbi_exe_path = read_bbi_path()), {
         # Note: all gradient colors will shift if number of models change
         c("#007319", "#FFFFFF", "#F4DBD3", "#ED9D84", "#E35B44", "#EB003D")
       )
+
+      # Regression test: numeric color_by is sorted appropriately
+      log_df <- run_log(MODEL_DIR)
+      log_df$numeric_vals <- c(1534, 3892, 731, 2653, 3574)
+      pl_tree <- model_tree(
+        log_df, add_summary = FALSE, color_by = "numeric_vals",
+        include_info = "numeric_vals"
+      )
+      node_colors <- get_node_attribute(pl_tree$x$data$children, attr = "fill")
+      expected_colors <- c("#F4DBD3", "#EB003D", "#FFFFFF", "#ED9D84", "#E35B44")
```

Comment:
I don't love embedding these values because, if I understand correctly, the values are calculated by another package (scales) and now our tests are tied to details of the scales calculation.  (I'm viewing it similar to tests matching against non-API messages from libraries with the expectation that they won't change, but perhaps this calculation is more set in stone.)

I'm fine with leaving this as is, and adjusting later if it turns out to be brittle.  (I see there are other tests in this file that already do this.)